xml/System.Security.Cryptography/AesGcm.xml

```diff
@@ -172,13 +196,24 @@
         <Parameter Name="associatedData" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="nonce">To be added.</param>
-        <param name="plaintext">To be added.</param>
-        <param name="ciphertext">To be added.</param>
-        <param name="tag">To be added.</param>
-        <param name="associatedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="nonce">The nonce associated with this message, which should be a unique value for every operation with the same key.</param>
+        <param name="plaintext">The content to encrypt.</param>
+        <param name="tag">The byte array to receive the genenerated authentication tag.</param>
+        <param name="ciphertext">The byte array to receive the encrypted contents.</param>
+        <param name="associatedData">Extra data associated with this message, which must also be provided during decryption.</param>
+        <summary>Encrypts the contents of <paramref name="plaintext"/> into <paramref name="ciphertext"/> and generates an authentication tag into <paramref name="tag"/>.</summary>
+        <remarks>The security guarantees of the AES-GCM algorithm mode require that the same nonce value is never used twice with the same key.</remarks>
```

Comment:
I noticed it's GCM, not CCM like in the two other suggestions above:

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

The security guarantees of the AES-GCM algorithm mode require that the same nonce value is never used twice with the same key.

          ]]></format>
        </remarks>
```

Reply:
> I noticed it's GCM, not CCM like in the two other suggestions above:

It should be the case that AesCcm references CCM and AesGcm references GCM. Assuming the "above" was for the other type/file, then everything is still correct :smile:.